### PR TITLE
Task 2 tic tac toe

### PR DESCRIPTION
**Pull request by Anameskhi**

### CODE REVIEW

File: week6_7/task_2/task_2_tic_tac_toe.rb [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+          count += 1 if i == "X" || i == "O"
+      end
+   count
+  end
+
+  def current_player
+    turn_count % 2 == 0 ? "X" : "O"
+  end
+
+  def turn
```
Comment (alexshvedko) [inline week6_7/task_2/task_2_tic_tac_toe.rb:61]: very long method

Comment (alexshvedko) [reply week6_7/task_2/task_2_tic_tac_toe.rb:61]: long method

File: week6_7/task_2/task_2_tic_tac_toe.rb [hunk 2]
```diff
@@ -29,59 +31,67 @@ def input_to_index(string)
...
 
   def move(input_to_index, player)
-    @board[input_to_index] = player
+      @board[input_to_index] = player
   end
 
   def position_taken?(input_to_index)
-    if (@board[input_to_index] == ' ') || (@board[input_to_index] == '') || @board[input_to_index].nil?
-      false
+    if (@board[input_to_index] == " ") || (@board[input_to_index] == "") || (@board[input_to_index] == nil)
```
Comment (neb5562) [inline week6_7/task_2/task_2_tic_tac_toe.rb:38]: `if ["", " ", nil].any? @board[input_to_index]`

File: week6_7/task_2/task_2_tic_tac_toe.rb (hunk 1 above)
Comment (Anameskhi) [reply week6_7/task_2/task_2_tic_tac_toe.rb:59]: Corrected! @alexshvedko

**Review by alexshvedko — APPROVED** [review]

**Pull request merged**